server/plugin.go

```diff
@@ -59,36 +86,59 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 		p.handleList(w, r)
 	case "/remove":
 		p.handleRemove(w, r)
+	case "/complete":
+		p.handleComplete(w, r)
+	case "/enqueue":
+		p.handleEnqueue(w, r)
+	case "/bump":
+		p.handleBump(w, r)
 	default:
 		http.NotFound(w, r)
 	}
 }
 
+type addAPIRequest struct {
+	Message string
```

Comment:
In order to be more informative in the code, we may add json tags, but they are not needed. From [json Go documentation](https://golang.org/pkg/encoding/json/#Unmarshal):

> To unmarshal JSON into a struct, Unmarshal matches incoming object keys to the keys used by Marshal (either the struct field name or its tag), preferring an exact match but also accepting a case-insensitive match.

Therefore, unless you want to make it more verbose, it should not be needed.

Reply:
Probably marginally faster if we use the tags and avoid case-insensitive comparison. I also personally like the tags highlight the JavaScript/JSON nature of the go type, typographically make it stand out in your face. But I'm 0/5 on this. Tags are visual clutter, too.

Reply:
Personally, I prefer to explicitly have the JSON tags because it reminds you that this is serialized. And as soon as you add a camel-case field you'll need to write the tag to turn it into json-style snake-case, so it reminds you that you need to do that.